### ISSUE TITLE
Test script fails

### ISSUE THREAD

**Issue by tillea** [CartwrightLab/dawg#56]
Hi,
if I'm calling the test script tests/test0.sh I get the following diff:

    2,3c2,3
    < TCCTTGACCAGTTAGCAAGACGATATGCATCAAGTGCACTGGC---GTAAGTCTTTTTAC
    < GCTGATCATA--TAGTCCGTATAGTCACTGAACGCCGTCCTCTCG
    ---
    > AGGAACTGGTCACGTTCTGCTATACGTAGTTCACGTGACCGCATTCAGAAAAATGCGACT
    > AGTATTGTGATCAGGCATATCAGTGACTTGCGGCAGGAGAGC
    6,7c6,7
    < TCGTTGGACAGT--GCAAGACGCTATGCATCAAGTGCACTGGCTAGGTAAGTGCGTTTAT
    < GATGAGCACAACAGGTCCGGATAGTCCCTGAACGCTATACTCCCG
    ---
    > AGCAACTTGTCACGTTCTGCGATACGTAGTTCACGTGACCGCATTCACGCAAACACTACT
    > TGTGCT--GACCATGCATATCAGGGACTTGCGACATGTGGGC
    10,11c10,11
    < TCGA-CCCCAAA--TTAATACGTTAGTCATCAAGTGCACTGAC---GTAATAGCGTTTAT
    < GATGATGAGTACTAGCCCGTGGAAACCCTAAAAGCGTTACCCCCG
    ---
    > AGCATGGTGTCAAGTTCTGCGTTCAGTAGTTAACCAGACGGCATTAACGCTAATACATC-
    > -GTGTT--GACCTACCAACGTACGGACTTGCGTAATGTGGTC
    14,15c14,15
    < TCGTTGAACAGT--GCAAGACGCTATGCATCAAGTGCACTGGC---GTAAGTGCGTTTAT
    < GATGAACACAACTGGTCCGTATAGTCCCTGAACGCTGTACTCCCG
    ---
    > AGCAACTTGTCACGTTCTGCGATACGTAGTTCACGTGACCGCATTCACGCAAATACTACT
    > TGTGTT--GACCAGGCATATCAGGGACTTGCGACATGAGGGC
    18,19c18,19
    < TCGTACCACAGT--TCAAGACGCTAGGCATCAAGTGCACTGGC---GTAATTGCGTTTAT
    < GATGGACACAACTAGTCCGTTGAATCCCTGAACGCTTTACCCCCG
    ---
    > AGCATGGTGTCAAGTTCTGCGATCCGTAGTTCACGTGACCGCATTAACGCAAATACTACC
    > TGTGTT--GATCAGGCAACTTAGGGACTTGCGAAATGGGGGC

I wonder what might be wrong here - dawg or the test itself?
Kind regards, Andreas.

**Comment by nileshpatra** [comment]
@reedacartwright could you answer this?

It also looks like `tests/test0.dawg` is incompatible with the current version of dawg (which outputs `dawg 2-current-rUnknown`) - could you please confirm this?

Additionally could you please tag a new GitHub release for 2.0? We ship dawg as a Debian package and it'd be really nice if we could have as less delta with the upstream (which is you :-))

**Comment by reedacartwright** [comment]
I don't believe that anyone has ever ported the test script to the 2.0 code base. 2.0 is still technically in development which is why I haven't tagged any releases. I will consider your request though.

**Comment by tillea** [comment]
Do you have any news?
It is not about this?  You talk about porting the test script to version 2.0 - but it fails for version 1.2 which is the latest tagged version.

**Comment by reedacartwright** [comment]
Honestly, cleaning up the tests for dawg is low on my priority list until development starts again. I'll review any patches made to improve its tests.
